components/drivers/audio/audio.c

```diff
@@ -135,7 +135,7 @@ static rt_err_t _aduio_replay_start(struct rt_audio_device *audio)
             result = audio->ops->start(audio, AUDIO_STREAM_REPLAY);
 
         audio->replay->activated = RT_TRUE;
-        LOG_D("start audio replay device");
+        LOG_E("start audio replay device");
```

Comment:
Why use LOG_E in here?

Reply:
Oh, It's my fault, this will be modification next time.